grumphp.yml

```diff
@@ -1,61 +0,0 @@
-parameters:
-    phpstan.config: "vendor/phpro/symfony-conventions/phpstan.neon"
-    phpstan.level: 2
-    phpcsfixer2.config: "vendor/phpro/symfony-conventions/.php_cs.dist"
-    phpparser.ignore:
-        - vendor
-        - config
-        - tests/bootstrap.php
-        - src/Kernel.php
-    phpstan.ignore:
-        - "tests/"
-        - "Tests/"
-        - "vendor/"
-        - "var/"
-        - "src/Migrations/"
-        - "spec/"
-grumphp:
-    stop_on_failure: true
-    tasks:
-        paratest:
-            metadata:
-                priority: 100
-        phplint:
-            metadata:
-                priority: 200
-        yamllint:
-            parse_custom_tags: true
-            metadata:
-                priority: 200
-        composer:
-            strict: false
-            no_check_publish: true
-            metadata:
-                priority: 200
-        phpparser:
```

Comment:
I'm thinking about unsupporting phpparser in the future. All the stuff in here can be enforced by cs tools as well + it can be quite slow on big applications. WDYT?